EgammaAnalysis/plugins/HGCalPhotonIDValueMapProducer.cc

```diff
@@ -67,6 +67,17 @@ HGCalPhotonIDValueMapProducer::HGCalPhotonIDValueMapProducer(const edm::Paramete
   maps_["sigmaVV"] = {};
   maps_["sigmaEE"] = {};
   maps_["sigmaPP"] = {};
+  maps_["nLayers"] = {};
+  maps_["firstLayer"] = {};
+  maps_["lastLayer"] = {};
+  maps_["firstLayerEnergy"] = {};
```

Comment:
Hello, 
sorry for the late reaction. I am not sure that "firstLayerEnergy" is a very good variable for ID. Layer per layer energies can fluctuate (especially in the first layer for photons), and I suspect it is quite sensitive to PU. 
F.